Author: Alastair McKinstry <mckinstry@debian.org>
Description: Disable test as website appears to be down
Forwarded: no
Last-Updated: 2015-10-20

Index: libdap-3.18.0/unit-tests/Makefile.am
===================================================================
--- libdap-3.18.0.orig/unit-tests/Makefile.am
+++ libdap-3.18.0/unit-tests/Makefile.am
@@ -67,9 +67,9 @@ D4-xml.tar.gz: D4-xml/DMR_*[0-9].xml
 if CPPUNIT
 UNIT_TESTS = marshT arrayT attrTableT structT sequenceT ddsT dasT \
 	RegexTest ArrayTest AttrTableTest ByteTest MIMEUtilTest ancT DASTest \
-	DDSTest	DDXParserTest  generalUtilTest HTTPConnectTest parserUtilTest \
-	RCReaderTest SequenceTest SignalHandlerTest  MarshallerTest \
-	HTTPCacheTest ServerFunctionsListUnitTest
+	DDSTest	DDXParserTest  generalUtilTest  parserUtilTest \
+	SequenceTest SignalHandlerTest  MarshallerTest \
+	ServerFunctionsListUnitTest
 
 if DAP4_DEFINED
 UNIT_TESTS += D4MarshallerTest D4UnMarshallerTest D4DimensionsTest \
